ecs-cli/modules/cli/local/local_app.go

```diff
@@ -44,3 +44,14 @@ func Up(c *cli.Context) {
 	}
 	network.Setup(docker)
 }
+
+// Stop stops a running local ECS task.
+//
+// If the user stops the last running task in the local network then also remove the network.
+func Stop(c *cli.Context) {
+	docker, err := client.NewEnvClient() // Temporary client created to test network.Teardown()
```

Comment:
You might want to consider doing something like I did here: https://github.com/awslabs/amazon-ecs-local-container-endpoints/blob/master/local-container-endpoints/clients/docker/client.go#L49

I once or twice ran into weird problems where the client API version was too new for my local docker. If you lock it to the min version that is needed, then customers are safe.